hadoop-pipeline-application/src/main/java/org/molgenis/hadoop/pipeline/application/HadoopPipelineApplication.java

```diff
@@ -44,12 +56,50 @@ public static void main(String[] args)
 	public int run(String[] args) throws IOException, ParseException, ClassNotFoundException, InterruptedException
 	{
 		Configuration conf = getConf();
+
+		// YARN fix for: java.lang.OutOfMemoryError: Java heap space
+		conf.set("mapreduce.map.memory.mb", "2048");
+		conf.set("mapreduce.map.java.opts", "-Xmx1024m");
+
```

Comment:
Update: The command line implementation should be present in the coming push (I think I figured out what went wrong last time I tried it, but didn't want to spend more time on it as getting the BwaAligner working was at that time a higher priority).